src/github/client/github-installation-client.ts

```diff
@@ -355,8 +356,12 @@ export class GitHubInstallationClient extends GitHubClient {
 		return response?.data?.data;
 	}
 
-	public async getDeploymentsPage(owner: string, repoName: string, perPage?: number, cursor?: string | number): Promise<getDeploymentsResponse> {
-		const response = await this.graphql<getDeploymentsResponse>(getDeploymentsQuery,
+	public async getDeploymentsPage(jiraHost: string, owner: string, repoName: string, perPage?: number, cursor?: string | number): Promise<getDeploymentsResponse> {
```

Comment:
Would the cursor always be a string in this case?

Reply:
Yup I think so, 
1. The typing I believe it is because we use rest api and was page no before? So typing to be backward compatibility?
2. And can confirm all deploymentCursor is string (not number as string) in prod db. I'll remember to remove this on ff cleanup.